src/components/Inputs/AudioInput/AudioInput.stories.mdx

```diff
@@ -38,8 +38,8 @@ It emits the audio in base64 format every time the user finish or pause the audi
 
 | Event                       | Description
 |:----------------------------|:---------------------------------------------------------------------
-| changeState                 | emit whenever the audio changes from recording to paused and paused to recording
-| audio                       | emit a base64 string whenever the audio buffer is created (when recording is paused)
+| change-state                | emit whenever the audio changes from recording to paused and paused to recording
+| audio                       | emit a object containing the audio blob whenever the audio buffer is created (when recording is paused) and the audio duration in seconds ({ data: audioBlob, duration: audioDuration })
```

Comment:
uses 'an' instead of 'a'
```suggestion
| audio                       | emit an object containing the audio blob whenever the audio buffer is created (when recording is paused) and the audio duration in seconds ({ data: audioBlob, duration: audioDuration })
```